docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -49,6 +53,12 @@ deactivate AddressBookParser
 LogicManager -> DeleteCommand : execute(m)
 activate DeleteCommand
 
+DeleteCommand -> Ui : showAlertDialogAndWait()
+activate Ui
+
+Ui --> DeleteCommand
+deactivate Ui
```

Comment:
For this sequence diagram, should we separate different components here?

Reply:
I will ask our TA if this is ok or is it considered a bug. Thanks for pointing it out!